docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,223 @@
+# Natural Language Processing (Nlp)
+
+
+Nlp is a subset of natural language toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (Nlp) functionalities, such as language detection, parts of speech, word toeknize, and entity detection. For more information, see the [nltk Forum](http://www.nltk.org/).
+
+The main features of the Tizen.Nlp namespace include:
+
+-   Word Tokenize suppport
+
+    You can get tokens from a sentence, the type of return is WordTokenizeResult.
+
+-   Part of Speech support
+
+    You can get tokens and tags from a sentence, the type of return is PosTagResult.
+
+-   Named Entity Detection support
+
+    You can get tokens and tags from a sentence, the type of return is NamedEntityRecognitionResult.
+
+-   Language Detect support
+
+    You can get language from a sentence, the type of return is LanguageDetectedResult.
+
+## Word Tokenize
+This method break up the sentence into words and punctuation.
```

Comment:
Suggestion: this part would be better if it is written in a tabular form.

![image](https://user-images.githubusercontent.com/37893210/46781864-417bc980-cd41-11e8-92d1-f86662a24355.png)

Reply:
Is it worth to create a table for it?
I think it is such a burden to manage "sentences" in a table which we already have a list about them from line 8.

As @divyk guided to remove link form list line 8 ~ line 22 to this sections,
how about just merging these sentences into the list above?

For example (This is just an example)


- Word Tokenize suppport
   You can get tokens from a sentence, the type of return is WordTokenizeResult. 
   This method break up the sentence into words and punctuation.

- Part of Speech support
   You can get tokens and tags from a sentence, the type of return is PosTagResult.
   This method breaks up the sentence into words and punctuation with tags attributes.
...